packages/storybook/src/assets/img/icons/check--grey.svg

```diff
@@ -0,0 +1,3 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 8">
```

Comment:
As mentioned above – I thin if you can add the icon to f-icons (or if there's n icon in there already that matches this one) then that would save you needing to duplicate the icon to storybook